QuickFIXn/SessionState.cs

```diff
@@ -402,7 +402,6 @@ protected virtual void Dispose(bool disposing)
             if (_disposed) return;
             if (disposing)
             {
-                Log.Dispose();
```

Comment:
@jkulubya What if `Log` is (e.g.) a `LogAdapter`, which is an `IDisposable`?

Although... SessionState doesn't create the Log object, so now I think it should never have been disposing it.  The containing Session object creates the log, and should probably be doing that disposal instead.

Is that anything you looked into or thought about while working on this?